staging/src/k8s.io/legacy-cloud-providers/openstack/metadata.go

```diff
@@ -53,6 +53,18 @@ const (
 
 	// configDriveID is used as an identifier on the metadata search order configuration.
 	configDriveID = "configDrive"
+
```

Comment:
Drop this commit and create BZ for node team instead.

Reply:
Done. https://bugzilla.redhat.com/show_bug.cgi?id=1814054